experiment_config/omniglot_maml-omniglot_1_8_0.1_64_5_0.json

```diff
@@ -37,6 +37,7 @@
   "weight_decay": 0.0,
   "dropout_rate_value":0.0,
   "min_learning_rate":0.00001,
+  "task_learning_rate":0.1,
```

Comment:
Was this missing in the original file? Does it cause a runtime error?

Reply:
Yes this was missing in the original json file. It does not cause any error. But, in your utils/parser_utils.py this is default to 0.1 (this might be useful I think)

Reply:
Basically, for MAML experiments with miniImagenet dataset, I wanted to set the inner loop learning rate to 0.01/0.001 and felt adding this to json will help. I think this should be added to all other .json files as well.